### PR TITLE
new icon: cloudrun (original, plain, line)

### LINKED ISSUE #2180: [ICON REQUEST]: Cloud Run
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

Cloud run is a development tool part of Google Cloud Platform

### Links and sources

Cloud Run is a fully managed platform that enables you to run your code directly on top of GCP
[Google's documentation](https://cloud.google.com/run) on Cloud Run 

# PNG 👇

cloudrun

[Link for SVG](https://www.svgrepo.com/svg/375383/cloud-run)

### Additional information

_N/A_

### PR DESCRIPTION
**Pull request by this-is-emma**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/wiki/Overview-on-Submitting-Icons)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/wiki/Organizing-SVGs)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/wiki/SVG-Standards)
- [x] A new object is added in the `devicon.json` file at the correct alphabetic position as seen [here](https://github.com/devicons/devicon/wiki/Updating-%60devicon.json%60)

**This PR closes #2180 **

## Link to prove your SVG is correct and up-to-date.
Google Tutorial on Cloudrun with [icon displayed](https://codelabs.developers.google.com/codelabs/cloud-starting-cloudrun-jobs#0)

### CODE REVIEW

**Review by canaleal — CHANGES_REQUESTED** [review]

File: icons/cloudrun/cloudrun-original.svg
[the remark this answers is not in the document]
Comment (this-is-emma) [reply icons/cloudrun/cloudrun-original.svg:1]: Thanks for the tutorial, this is will be helpful!

File: icons/cloudrun/cloudrun-plain.svg
Comment (canaleal) [inline icons/cloudrun/cloudrun-plain.svg:1]: In this SVG the paths have to be unified into a single path and optimized just like the original.svg

File: icons/cloudrun/cloudrun-line.svg
Comment (canaleal) [inline icons/cloudrun/cloudrun-line.svg:1]: In this SVG the paths have to be unified into a single path and optimized just like the original.svg.

**Review by canaleal — CHANGES_REQUESTED** [review]
Nice work. We just need to make the sure that the SVGs are optimized and that the plain and line svgs only have 1 path.

**Review by canaleal — APPROVED** [review]
It's been some time so I decided to optimize the SVGs using SVGOMG. 

No artifacts on icomoon:

image

**Comment by canaleal** [comment]
All issues have been resolved and there are no artifacts in the SVGs.

image

**Pull request merged**